### ISSUE TITLE
Nifity Bug

### ISSUE THREAD

**Issue by joseJCS0** [ContextLab/supereeg#213]
When trying to load in the Nifity example file as instructed in the tutorial an error occurred resulting in "HeaderDataError: data code 26996 not recognized".Locally could not/has not been resolved will attach the full error output.
[nifty_bug.txt](https://github.com/user-attachments/files/17168555/nifty_bug.txt)

**Comment by jeremymanning** [comment]
@joseJCS0 are you able to share with us a sample of the data you're using here so that we can reproduce the issue?  (Ideally a small sample!)

**Comment by lucywowen** [comment]
hey @jeremymanning, @joseJCS0 is a potential new masters student in my lab and he's getting familiar with supereeg right now.  I think the nii example file he's referring to is the one that on google docs (the one that's loaded in load.py).

**Comment by jeremymanning** [comment]
oh, that's awesome!  @joseJCS0, welcome!  in that case, @lucywowen do you want to go through these updates with him?  let me know if/how i can be of use, or i'm also happy to just observe :)

**Comment by lucywowen** [comment]
Yes absolutely!  Thanks @jeremymanning!  I'm going to walk through all the bugs... but we might need your help with this one. I don't think I have access to the google docs with the example nifti
